data/kammy1996.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Kamran Memon",
+  "type": "personal",
+  "bio": "I am a frontend Developer working with Vue.js",
+  "avatar" : "https://github.com/kammy1996.png",
```

Comment:
hey @kammy1996 please delete this `avatar` code. Currently, you don't need to add this command. 
``` suggestion
 
```

Reply:
Changes done as per suggestion